modules/rules_java/7.0.1/MODULE.bazel

```diff
@@ -0,0 +1,78 @@
+module(
+    name = "rules_java",
+    version = "7.0.1",
+    compatibility_level = 1,
```

Comment:
```suggestion
    bazel_compatibility = [">=6.4.0"],
    compatibility_level = 1,
```

Reply:
Ack, fixing

Reply:
This is missing https://github.com/bazelbuild/rules_java/pull/143.

Reply:
you can close this one in favour of https://github.com/bazelbuild/bazel-central-registry/pull/1024